osp/core/session/core_session.py

```diff
@@ -71,10 +72,10 @@ def __len__(self):
     class CoreSessionSparqlBindingSet(SparqlBindingSet):
         """A row in the result. Mapping from variable to value."""
 
-        def __init__(self, row, session):
+        def __init__(self, row, session, datatypes):
```

Comment:
Could you make `datatypes` a keyword argument with default value `None` here as you do in `SparqlBindingSet`? Here it's useless, but if any other code that uses the `CoreSessionSparqlBindingSet` is added, then it would be less verbose.